apps/vs-code-designer/src/app/commands/deploy/deploy.ts

```diff
@@ -216,14 +219,21 @@ async function getDeployLogicAppNode(context: IActionContext): Promise<SlotTreeI
   const placeHolder: string = localize('selectLogicApp', 'Select Logic App (Standard) in Azure');
   const sub = await ext.rgApi.appResourceTree.showTreeItemPicker<AzExtParentTreeItem>(SubscriptionTreeItem.contextValue, context);
 
-  const [site, isAdvance] = (await context.ui.showQuickPick(getLogicAppsPicks(context, sub.subscription), { placeHolder })).data;
+  let [site, isAdvance] = (await context.ui.showQuickPick(getLogicAppsPicks(context, sub.subscription), { placeHolder })).data;
   if (!site) {
     if (isAdvance) {
       return await createLogicAppAdvanced(context, sub);
     }
     return await createLogicApp(context, sub);
   }
+
+  if (site.id.includes('Microsoft.App')) {
```

Comment:
is this the same as `isHybridLogicApp`?

Reply:
yes, but this point we do not have the `isHybridLogicApp` parameter assigned